modules/navigation.js

```diff
@@ -0,0 +1,95 @@
+export default () => {
+  const form = document.querySelector('#form');
+  const nav = document.querySelector('.nav');
+  const close = document.querySelector('.close');
+  const navUl = document.querySelector('.navUl');
+  const container = document.querySelector('.container');
+  const navContainer = document.querySelector('.navContainer');
+  const title1 = document.querySelector('.title1');
+  const navLi = document.querySelectorAll('.navLi');
+  const contactInfoDiv = document.querySelector('.contactInfoDiv');
+
+  const navOpen = () => {
+    container.classList.add('d-none');
+    form.classList.add('d-none');
+    title1.classList.add('d-none');
+    navContainer.classList.add('navActive');
+    navContainer.classList.remove('navContainer');
+    nav.style.display = 'none';
+    close.classList.add('closeActive');
+    navUl.style.display = 'flex';
+    navUl.classList.add('navUlActive');
+    navLi.forEach((i) => {
+      i.classList.add('navLiActive');
+    });
+  };
+
+  const navClose = () => {
+    container.classList.remove('d-none');
+    form.classList.remove('d-none');
+    title1.classList.remove('d-none');
+    navContainer.classList.remove('navActive');
+    navContainer.classList.add('navContainer');
+    close.classList.remove('closeActive');
+    navUl.style.display = 'none';
+    navUl.classList.remove('navUlActive');
+    nav.style.display = 'block';
+    contactInfoDiv.style.display = 'none';
+  };
+
+  const navClose2 = () => {
+    container.classList.remove('d-none');
+    form.classList.remove('d-none');
+    title1.classList.remove('d-none');
```

Comment:
[OPTIONAL] Your logic is working perfectly, well done! Just a friendly piece of advice, instead of naming your functions `navClose` and `navClose2` try using something more descriptive, this way any other developer will understand easily the difference between both functions.